src/lib/api/containerApiClient.test.ts

```diff
@@ -0,0 +1,80 @@
+import { fetchAllContainerList, fetchContainerList } from '@/lib/api';
+import * as commonUtils from '@/lib/api/commonUtils';
```

Comment:
Actually, I wanted to avoid this too and I tried not to use `*` with many patterns but It throw something error, so that's why I used it.
It is a test code not actual source code, so I thought it is allowed... But what do you think?

Reply:
The following code is my suggestion. It should work fine.

I'm also a bit concerned about calling `mockRejectedValue()` individually, even though `setUpMockRequest()` is defined for `mockResolvedValue()`. Aligning them would be more consistent.

Here's the code:
```ts
import { request } from '@/lib/api/commonUtils';
:
const setUpMockRequest = <T>(mockData: T) => {
  return (request as jest.MockedFunction<typeof request>).mockResolvedValueOnce(mockData);
};
:
describe('API Function Tests', () => {
  :
  describe('fetchContainerList', () => {
    :
    it('throws an error on API failure', async () => {
      (request as jest.MockedFunction<typeof request>).mockRejectedValue(
        new Error('Network error'),
      );
      :
    });
    :
  });
  :
});
```

Reply:
@kotaaaa 
By the way,

> It is a test code not actual source code, so I thought it is allowed... But what do you think?

I understand your perspective, but I believe that all the code currently in use should adhere to our guidelines, especially in open-source projects where external contributors could participate. Even though we have provided a style guide, it's likely that contributors will try to follow the existing code's formatting. If we need to deviate from the usual formatting, we should provide a clear explanation for that particular instance as a comment.

Reply:
Thank you for your suggestion! It works! I fixed the code. Please check them.

Reply:
> I understand your perspective, but I believe that all the code currently in use should adhere to our guidelines, especially in open-source projects where external contributors could participate. Even though we have provided a style guide, it's likely that contributors will try to follow the existing code's formatting. If we need to deviate from the usual formatting, we should provide a clear explanation for that particular instance as a comment.

Understood. I am following it.

Reply:
@kotaaaa Thank you for the update! 

> I'm also a bit concerned about calling mockRejectedValue() individually, even though setUpMockRequest() is defined for mockResolvedValue(). Aligning them would be more consistent.

May I also hear your thoughts on this?

Reply:
@uskayyyyy 
> I'm also a bit concerned about calling mockRejectedValue() individually, even though setUpMockRequest() is defined for mockResolvedValue(). Aligning them would be more consistent.

Thanks, I refactored test codes. Please check them.